ui/src/app/utils/access-utils.tsx

```diff
@@ -486,6 +486,31 @@ export const isExpiringOrExpired = (
   return !!expiration && getWholeDaysFromNow(expiration) <= lookback;
 };
 
+// is the module "renewal complete" ?
+// meaning (bypassed || (complete and not expiring))
+export const isRenewalCompleteForModule = (status: AccessModuleStatus) => {
+  const isComplete = !!status?.completionEpochMillis;
+  const wasBypassed = !!status?.bypassEpochMillis;
+  return (
+    wasBypassed ||
+    (isComplete &&
+      !isExpiringOrExpired(status?.expirationEpochMillis, status.moduleName))
+  );
+};
+
+export const isRtRenewalComplete = (profile: Profile): boolean => {
+  const modules = profile?.accessModules?.modules;
```

Comment:
is there a better alternative?  I could do something like `const {modules} = profile?.accessModules ?? {}` but that doesn't really seem to be an improvement